README.md

```diff
@@ -1,12 +1,15 @@
 # Music Releases
-Replace this readme with your own information about your project.
+Our assignment was to create components using data in React to build a website that displays the latest music releases from Spotify. We were to follow a given design and make the site responsive accordingly.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
 
 ## The problem
+We had an API-fetch from Spotify to start with, and we began with displaying the image, artist name and album name using that API. We also made the artist name and the album name lead to an external links to their spotify-pages. 
+
+The album and artist information were put into a separat component. We also had the header in a separat component. In our App.js we imported all components to make the website complete.
+
+For styling we used grid to make the website responsive. We followed the given design using CSS and added hover-effects and adding "," and "&" between the artists names. We had issues with the overlay showing when the site was deployed. A classmate gave us the tip to change from opacity to transform:scale() to hide/show the overlay, which worked perfectly.
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://modest-torvalds-b6767e.netlify.app/
```

Comment:
On netlify under site settings it's possible to change the url to a custom name if that is something of interest! :) Easier in the future to search for the site as well!